addons/stock_dropshipping/models/sale.py

```diff
@@ -15,7 +15,8 @@ def _get_qty_procurement(self, previous_product_uom_qty):
         if purchase_lines_sudo.filtered(lambda r: r.state != 'cancel'):
             qty = 0.0
             for po_line in purchase_lines_sudo.filtered(lambda r: r.state != 'cancel'):
-                qty += po_line.product_uom._compute_quantity(po_line.product_qty, self.product_uom, rounding_method='HALF-UP')
+                amount = tools.float_round(po_line.product_qty, precision_rounding=po_line.product_uom.rounding, rounding_method='HALF-UP')
```

Comment:
`tools.float_round() --> fields.Float.round()`

Reply:
@amoyaux can you check this PR
Note : in POL product_qty is the quantity in the UoM of POL (it is the inverse of SOL)